solidity/dashboard/src/sagas/utils.js

```diff
@@ -108,6 +110,7 @@ export function* subscribeToEventAndEmitData(
         : action(event)
       yield put(_action)
     } catch (error) {
+      console.log("error", error)
       console.error(`Failed subscribing to ${_debugName} event`, error)
```

Comment:
Do we need `console.log` here since we have `console.error`?

Reply:
c64211f